### PR TITLE
📃 Allow specifying grid to load from cmd line arg in bevy example

### PR DESCRIPTION
**Pull request by EmilioLaiso**
As the title says.
Not all VDB files have a "density" grid, and loading them should still be possible without having an hardcoded grid name.
This goes back to the original behaviour of loading the first grid found if none is specified in the cmd line, and will print the available ones in that case so that the user can make a different choice based on the VDB content if needed.

### CODE REVIEW

File: examples/bevy.rs [hunk 1]
```diff
@@ -51,7 +51,17 @@ fn setup(
...
+
+    let grid_to_load = std::env::args().nth(2).unwrap_or_else(|| {
+        println!(
+            "Grid name not specified, defaulting to first available grid.\nAvailable grids: {:?}",
+            grid_names
+        );
+        grid_names.first().cloned().unwrap_or(String::new())
+    });
+
+    let grid = vdb_reader.read_grid::<half::f16>(&grid_to_load).unwrap();
```
Comment (EmilioLaiso) [inline examples/bevy.rs:64]: Actually, there's still this assumption here. Will have to figure this out 😬

File: src/reader.rs [hunk 2]
```diff
@@ -155,10 +155,10 @@ impl<R: Read + Seek> VdbReader<R> {
 
     pub fn read_grid<ExpectedTy: Pod>(
         &mut self,
-        name: &'static str,
+        name: &str,
     ) -> Result<Grid<ExpectedTy>, ParseError> {
         let grid_descriptor = self.grid_descriptors.get(name).cloned();
-        let gd = grid_descriptor.ok_or(ParseError::InvalidGridName(name))?;
+        let gd = grid_descriptor.ok_or(ParseError::InvalidGridName(name.to_owned()))?;
```
Comment (MarijnS95) [inline src/reader.rs:161]: Doesn't clippy complain that this should be:

```suggestion
        let gd = grid_descriptor.ok_or_else(|| ParseError::InvalidGridName(name.to_owned()))?;
```

?

**Review by MarijnS95 — APPROVED** [review]

File: src/reader.rs (hunk 2 above)
Comment (EmilioLaiso) [reply src/reader.rs]: No but let's do it anyway :)

**Review by MarijnS95 — APPROVED** [review]

File: examples/bevy.rs (hunk 1 above)
Comment (Jasper-Bekkers) [reply examples/bevy.rs:64]: I think this assumption is fine for the api, though maybe in the sample you want to get a bit more fancy (probably not,it's just a sample). The format type is tied directly to the type of grid (eg. for velocity you'd expect a vec3 or something, for density an f32 etc).

Comment (MarijnS95) [reply examples/bevy.rs:64]: By that time you might want to switch to `clap` too - then this almost becomes a publishable binary executable with the crate as a Rust-written VDB viewer. Simple and elegant.

**Pull request merged**